### ISSUE TITLE
라이센스를 넣어야하는데 넣지를 않았네요 하하.

### ISSUE THREAD

**Issue by benjioh5** [benjioh5/CS-Univ-Wiki#68]
컨트리뷰터가 8분이나 있는 상황에서 라이센스 생각을 못하다니...
는 CC BY-SA나 CC BY-NC-SA 혹은 MIT 생각 중인데 어떻게 생각하시는지요?

**Comment by benjioh5** [comment]
일단 결정 나면, 컨트리뷰터들에게 다 허락 받고 바꿔야겠죠. 아악... 귀찮아진다.....

**Comment by miaekim** [comment]
뭐가뭔지모르겠는데

**Comment by benjioh5** [comment]
@miaekim CC는 나무-엔하-위키피디아에서 쓰는 것, MIT는... 알잖아...

**Comment by miaekim** [comment]
아니 다른 라이선스도 있는데 굳이 MIT만 언급한점ㅁ 왜  뭐때문에 /  그리고 나무랑 위키피디아랑  호환 안되는걸로아는데 흠 그래서 헷갈림

**Comment by benjioh5** [comment]
나무가 CC 2.0, 위키는 CC 3.0이니까. MIT는 그냥 제일 무난해보여서.........

**Comment by miaekim** [comment]
별일 없으면 위키피디아랑 호환되는 라이선스였으면 좋겠어.  필요에따라 공동기재도 되고 글도 긁어올 수 있도록

**Comment by benjioh5** [comment]
사실 여기저기에 글 쓰는 건 저작권적 문제가 별로 없음. 한 사람이 나무위키나 위키피디아에 같이 써도 그 사람 저작권은 유지가 되는거고, 쓴 글에 대해서 수정이나 재배포 해도 된다고 했으니 딴 사람이 덧붙여서 쓸 수 있는 것 뿐이지. 특히 CC는 저작권을 포기하는 라이센스가 아니라 저작물을 공유할 때 쓰는 라이센스기도 하고..... 뭐 퍼오기를 하고 싶다면 CC3.0도 나쁘진 않지.

**Comment by ardiefox** [comment]
CC BY-SA에 한 표 냅니다. 굳이 NC로 제한을 걸고 싶지는 않아요.

'하고 싶으면 해라'가 BSD 정신이니까요.(?)
